netbsd/README.md

```diff
@@ -0,0 +1,148 @@
+# NetBSD fwupd port notes
+
+## Table of contents
+
+- [NetBSD fwupd port notes](#netbsd-fwupd-port-notes)
+  - [Table of contents](#table-of-contents)
+  - [Running NetBSD in QEMU](#running-netbsd-in-qemu)
+  - [Customising the kernel](#customising-the-kernel)
+    - [Prerequisites](#prerequisites)
+    - [Building the kernel](#building-the-kernel)
+    - [Running a virtual machine with a custom kernel](#running-a-virtual-machine-with-a-custom-kernel)
+    - [NetBSD ESRT recognition - implementation proposition](#netbsd-esrt-recognition---implementation-proposition)
+
+## Running NetBSD in QEMU
+
+Wiki entry:
+- https://wiki.qemu.org/Hosts/BSD#NetBSD
+
+NetBSD users mailing list might contain some info regarding the installation 
+process:
+- http://mail-index.netbsd.org/netbsd-users/
+
+## Customising the kernel
+
+Thorough documentation and instructions:
+- http://netbsd.org/docs/kernel/
+
+### Prerequisites
```

Comment:
@Silentsky0 lets clarify here (and section below) that building is done on NetBSD started in qemu

Reply:
the building is not done on NetBSD running on qemu, I added information that we use crosscompilation